build.gradle.kts

```diff
@@ -141,11 +149,15 @@ spotless {
 tasks.check { dependsOn(tasks.spotlessApply) }
 
 tasks.withType<JavaCompile>().configureEach {
-    options.compilerArgs.addAll(listOf("-Xlint:all", "-Werror"))
+    options.compilerArgs.addAll(
+        listOf("-Xlint:all", "-Xlint:-requires-automatic", "-Xlint:-requires-transitive-automatic", "-Werror"))
     when (this) {
         tasks.compileJava.get() ->
             options.errorprone {
                 disableWarningsInGeneratedCode = true
+                // Error Prone does not understand the `@hidden` standard tag
+                disable("InvalidBlockTag")
+                disable("AssignmentExpression")
```

Comment:
I upgraded the Error Prone, and it now has the `AssignmentExpression` check enabled by default. We don't have a problem with people using such expressions lightly, but they can be useful in some situations. We already have a single instance of using an assignment expression in the codebase.